docs/source/trude.rst

```diff
@@ -1,5 +1,111 @@
+
+.. _Trude input:
+
+Input
+-----
+
+ * ``/Run/events``: list of the ``evt_number`` and the correspondent ``timestamp``.
+ * ``/Run/runInfo``: stores the ``run_number``.
+ * ``/RD/sipmrwf``: stores the raw waveform itself. It indicates the number of :math:`ADCs` per time bin for each event and SiPM.
+
+.. _Trude output:
+
+Output
+------
+
+* ``/HIST/sipm_dark``: histogram values of the dark spectrum.
+* ``/HIST/sipm_dark_bins``: bin  values of the dark spectrum.
+* ``/HIST/sipm_spe``: histogram values of the light spectrum.
+* ``/HIST/sipm_spe_bins``: bin  values of the light spectrum.
+
+.. _Trude config:
+
+Config
+------
+
+Besides the :ref:`Common arguments to every city`, *Trude* has the following arguments:
+
+.. list-table::
+   :widths: 50 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``min|max_bin``
+     - ``float``
+     - Lower/upper limit of the number of :math:`ADCs` of the waveform to be considered for the spectrum.
+
+   * - ``bin_width``
+     - ``int``
+     - Width of the bins for the spectrum.
+
+   * - ``proc_mode``
+     - ``string``
+     - Two options of subtracting the baseline: ``subtract_mode`` to subtract the mode and ``subtract_median`` to subtract the median.
+
+   * - ``number_integrals``
+     - ``int``
+     - Number of integrals to be performed in the waveform's time window. Typically this value is selected depending on the number of LED pulses in the waveform. E.g. if the waveform has 5 pulses, the number of integrals will be 5.
+
+   * - ``integral_start``
+     - ``int``
+     - Bin where the first integral of the set of ``number_integrals`` starts. The other integrals are performed using ``integrals_period`` assuming the same distance between the peaks.
+
+   * - ``integral_width``
+     - ``int``
+     - Number of bins to be considered for the integral starting at ``integral_start``. With ``integral_start`` these values are ideally chosen to select the whole pulse of the LED.
+
+   * - ``integrals_period``
+     - ``int``
+     - Period between integrals. This parameter won't be considered when ``number_integrals = 1``.
+
+
+.. _Trude workflow:
+
+Workflow
+--------
+
+Trude performs the following operations to obtain the spectra:
+
+ * :ref:`Selection of integration limits <Selection of integration limits>`
+ * :ref:`SiPM waveforms baseline subtraction <SiPM waveforms baseline subtraction>`
+ * :ref:`Spectrum histogram <Spectrum histogram>`
+
+
+.. _Selection of integration limits:
+
+Selection of integration limits
+::::::::::::::::::::::::::::::::
+
+Given the config parameters, the first step in this city would be to select the dark and light bins to compute the dark and light spectra.
+
+The light bins will be centered in the light pulse(s) and the dark bins are chosen as an interval of ``integral_width`` bins which ends 2 microseconds before the light pulse begins, as shown here:
+
+ .. image:: images/trude/wf_intervals.png
```

Comment:
I think it will help when reading the documentation to have a better explanation of the parameters defined before and to which range, etc on the spectra correspond. You can check [Buffy](https://next-exp-sw.readthedocs.io/en/latest/buffy.html) documentation as a reference.